draft-ietf-httpbis-http2bis.xml

```diff
@@ -262,21 +262,26 @@
     <section anchor="starting">
       <name>Starting HTTP/2</name>
       <t>
-        An HTTP/2 connection is an application-layer protocol running on top of a TCP connection
-        (<xref target="TCP"/>). The client is the TCP connection initiator.
+        Implementations that generate HTTP requests need to discover whether a server supports
+        HTTP/2.
       </t>
       <t>
-        HTTP/2 uses the "http" and "https" URI schemes defined in <xref target="HTTP" section="4.2"/>, with the same default port numbers.  As a result,
-        implementations processing requests for target resource URIs like <tt>http://example.org/foo</tt> or <tt>https://example.com/bar</tt> are required to first discover whether the
-        upstream server (the immediate peer to which the client wishes to establish a connection)
-        supports HTTP/2.
+        HTTP/2 uses the "http" and "https" URI schemes defined in <xref target="HTTP"
+        section="4.2"/>, with the same default port numbers as HTTP/1.1 (<xref
+        target="HTTP11"/>). These URIs do not include any indication about what versions an upstream
+        server (the immediate peer to which the client wishes to establish a connection) supports
+        HTTP/2.
```

Comment:
The last sentence here isn't right: it reads: "These URIs do not include any indication about what versions an upstream server supports HTTP/2".

Consider:

- "These URIs do not include any indication about what HTTP versions an upstream server supports."